source/_components/cover.rflink.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: "RFLink Cover"
+description: "Instructions how to integrate RFLink Somfy RTS Cover into Home Assistant."
+date: 2017-08-08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rflink.png
+ha_category: Cover
+ha_release: 0.50
```

Comment:
Release will need to be updated before merge. Most likely to 0.52 since 0.51 is rather imminent.